synapse/models/inet.py

```diff
@@ -1544,7 +1544,7 @@ def getModelDefs(self):
 
                     ('inet:service:login:method:taxonomy', ('taxonomy', {}), {
                         'interfaces': ('meta:taxonomy',),
```

Comment:
This is another of the few places we don't use "type" (e.g. inet:service:login:method:taxonomy vs inet:service:login:method:type:taxonomy - which is icky long - or just inet:service:login:type:taxonomy.

Can see "method" being more intuitive here but wanted us to decide vs oversight. :-)